binary/__init__.py

```diff
@@ -95,6 +95,7 @@
     'MemoryTierConfig',
     'DiskTierConfig',
     'TieredStoreConfig',
+    "PartitioningAttributeConfig",
```

Comment:
Minor: I believe this is not needed for now. Maybe we can remove references to non-list field from this PR